lesson24/features/pageobjects/Base.ts

```diff
@@ -0,0 +1,19 @@
+export class Base {
+
+    async isContainingInUrl(text: string) {
+        await browser.waitUntil(
+            async () => (await browser.getUrl()).includes(text),
+            {
+                timeout: 3000,
+                timeoutMsg: 'The Page loads more then 5 sec(',
+            }
+        );
+        return (await browser.getUrl()).includes(text);
+    }
+
+    async isElementVisible(locator: string): Promise<boolean> {
```

Comment:
this method can only return true, I do not this that this method make sense in this view